tests/test_matrix_utils.py

```
@@ -5696,6 +5712,88 @@ async def test_login_matrix_bot_login_response_status_codes(
+    mock_main_client.close = AsyncMock()
+
+    with (
+        patch("mmrelay.config.load_config", return_value={}),
+        patch("mmrelay.config.is_e2ee_enabled", return_value=False),
+        patch("mmrelay.matrix_utils.os.path.exists", return_value=False),
+        patch("mmrelay.matrix_utils.logger") as mock_logger,
+    ):
+        result = await login_matrix_bot(
+            homeserver="https://matrix.org",
+            username="user",
+            password="pass",
```

Ruff (0.15.7)

[error] Possible hardcoded password assigned to argument: "password"

(S106)